### DISCUSSION TITLE
Some influence to grow larger?

### DISCUSSION THREAD

**Discussion by andylo17** [MaxRobinsonTheGreat/EvolutionSimulatorV2#161] [Ideas and Suggestions]
as of right now, i have messed with this simulator countless times. I have allowed it to freely evolve for hours and yet every time it is the small, 2 celled hunters with a single mover and mouth that come out triumphant. Each generation of any organism whether stationary or moving will tend to shrink as time goes on, losing complexity, and resorting to the most basic form with the most efficient way of accessing food rather than worrying about protection or inflicting harm.

**Comment by theomgdev** [comment]
Decrease lifespan over time. Life time is something like lifespan*creature_size so bigger creatures lives longer and smaller creatures dies sooner. Largest recorded creature of the LifeEngine(aka Bob) created using this method.
